ci/release/update-version.sh

```diff
@@ -41,6 +41,8 @@ function sed_runner() {
 # CMakeLists update
 sed_runner 's/'"CUGRAPH VERSION .* LANGUAGES C CXX CUDA)"'/'"CUGRAPH VERSION ${NEXT_FULL_TAG} LANGUAGES C CXX CUDA)"'/g' cpp/CMakeLists.txt
 sed_runner 's/'"branch-.*\/RAPIDS.cmake"'/'"branch-${NEXT_SHORT_TAG}\/RAPIDS.cmake"'/g' cpp/CMakeLists.txt
+sed_runner 's/'"CUGRAPH_ETL VERSION .* LANGUAGES C CXX CUDA)"'/'"CUGRAPH_ETL VERSION ${NEXT_FULL_TAG} LANGUAGES C CXX CUDA)"'/g' cpp/libcugraph_etl/CMakeLists.txt
+sed_runner 's/'"branch-.*\/RAPIDS.cmake"'/'"branch-${NEXT_SHORT_TAG}\/RAPIDS.cmake"'/g' cpp/libcugraph_etl/CMakeLists.txt
```

Comment:
```suggestion
sed_runner 's|'"branch-.*/RAPIDS.cmake"'|'"branch-${NEXT_SHORT_TAG}/RAPIDS.cmake"'|g' cpp/libcugraph_etl/CMakeLists.txt
```

FYI, you can use other characters besides `/` as a separator  (i.e. `|`) in `sed` commands to avoid having to escape `/` characters in your patterns. We do this on the last line of this file as well ([link](https://github.com/rapidsai/cugraph/blob/3b37fe7628dcfd93f2ddb48871d79509915398bb/ci/release/update-version.sh#L60)).

([src](https://www.gnu.org/software/sed/manual/sed.html#The-_0022s_0022-Command), see screenshot)

![image](https://user-images.githubusercontent.com/7400326/143927163-d50be702-9d54-4c24-b10b-e3c1aa6157f3.png)